client/src/components/ActiveChat/Messages.js

```diff
@@ -5,12 +5,14 @@ import moment from 'moment';
 
 const Messages = (props) => {
   const { messages, otherUser, userId } = props;
+  const messagesNum = messages.length;
 
   return (
     <Box>
-      {messages.map((message) => {
+      {messages.map((_ , index) => {
+        const message = messages.at(messagesNum - index - 1);
```

Comment:
While this technically works to fix the order of the messages, it has some efficiency concerns because now you'll have to add new messages to the beginning of the messages array, which is an O(n) operation. Ideally the messages will already be sorted by the time it reaches the return statement. It would be better to handle the logic of reversing the messages either when we get the data from the API or, even better, in the back-end 👍🏻